Chetna_Mongmaw/UsersApplication/UsersApplication/Models/WebAppContext.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace UsersApplication.Models
+{
+    public partial class WebAppContext : DbContext
+    {
+        public WebAppContext()
+        {
+        }
+
+        public WebAppContext(DbContextOptions<WebAppContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Users> Users { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
+                optionsBuilder.UseSqlServer("Server=CYG317\\SQLEXPRESS;Database=WebApp;Trusted_Connection=True;");
```

Comment:
Please  Explain the use of this connection string

Reply:
corrected